app/assets/html/sidebar.html

```diff
@@ -1,4 +1,67 @@
+<!-- This template is deprecated for now but leave for debugging
+- <script type="text/ng-template" id="requests_content.html">
+-     <div class="fetched-users-container">
+-         <li ng-repeat="matchedGladiator in matchedGladiators">
+-             <div class="fetched-user">
+-                 <a class="fetched-user-name">{{ matchedGladiator }}</a>
+-                 <a>{{ testvar }}</a>
+-             </div>
+-         </li>
+-     </div>
+- </script>
+-->
+<!-- Add this later
+    <a class="btn btn-primary send-request" ng-click="request_match(request.id)">Request</a>
+-->
+
+<!-- TODO: General
+    - CSS Makeover
+    - Hook up to API calls on backend
+-->
+
+<!-- TODO: Requests Popover
+    - Fix ng-repeat and make "matchedGladiator in matchedGladiators" work
+    - Add a "Request" button to every match
+    - Fix Popover clipping
+    - Make button press work
+    - Fix trigger to close when clicking outside
+-->
+<div class="requests-menu" ng-controller="SidebarController">
+    <button type="button" ng-model="singleModel" btn-checkbox btn-checkbox-true="1" btn-checkbox-false="0" requests-popover title="Send requests to be matched with other gladiators">Find Gladiators!</button>
```

Comment:
Implementation: The popup displays the Coffeescript list rather than the usernames/links to profiles.  
CSS/Style: button isn't styled, also consider changing verbiage to "Find Another Gladiator!" or something similar?